### DISCUSSION TITLE
Can u help me to fix this

### DISCUSSION THREAD

**Discussion by Akkimdivya** [community/community#47882] [GitHub Education] [answered]
You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

**Comment by melvincwng** [comment, marked as answer]
Hey @Akkimdivya, for issues regarding the application of GitHub student developer pack, you can raise a support ticket over [here](https://support.github.com/contact/education).

It usually takes a few days to weeks for a support engineer at GitHub to get back to you.

Hope this helps resolves your issue, and answers your question 😃

**Reply by samus-aran** [reply]
We appreciate your assistance @melvincwng . 👍  This type of inquiry we would not recommend directing to the support team as they are unable to provide manual reviews.

**Comment by samus-aran** [comment]
Hi @Akkimdivya 👋 

We're happy you're here. Let's see what we can do to assist.

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
